src/pudl/analysis/plant_parts_eia.py

```diff
@@ -365,6 +366,55 @@
 ]
 
 
+def plant_parts_eia_asset_factory(
+    io_manager_key: str | None = None,
+) -> list[AssetsDefinition]:
+    """Build EIA plant parts table related assets."""
+
+    @asset(
+        name="mega_generators_eia",
+        ins={
+            "mcoe": AssetIn(key="mcoe_generators_yearly"),
+            "own_eia860": AssetIn(key="denorm_ownership_eia860"),
+        },
+        io_manager_key=None,
```

Comment:
Is this all that needs to happen so that this table isn't written to the DB?

Reply:
There's a default IO manager which will be used if this is None, which could be set explicitly instead.  But unless it's set to `pudl_sqlite_io_manager` it won't be written to the DB.